azimuth/dataset_split_manager.py

```diff
@@ -153,6 +152,21 @@ def _load_dataset_split(self) -> Optional[Tuple[Dataset, Dataset]]:
             return ds, malformed
         return None
 
+    def _load_base_dataset_split(self, dataset_split) -> Tuple[Dataset, Dataset]:
+        base_dataset_split, malformed_dataset = self._split_malformed(dataset_split)
+
+        # Checking if a persistent id was provided.
+        persistent_id = self.config.columns.persistent_id
+        if persistent_id != DatasetColumn.row_idx:  # Default value
+            if persistent_id not in base_dataset_split.column_names:
+                raise ValueError(f"Persistent ID named {persistent_id} not found in the dataset.")
```

Comment:
I know not everyone is super rigorous on this, but I like to make the distinction:
- lowercase "id" is an abbreviation for "identifier". That's what we have here.
- uppercase "ID" is an acronym, so I reserve it for "Identity Document". That is a document like a passport or a driver's license.

Does that make sense to you?
```suggestion
                raise ValueError(f"Persistent id named {persistent_id} not found in the dataset.")
```

Reply:
Totally! Thank you!